library/src/com/google/maps/android/geojson/GeoJsonPoint.java

```diff
@@ -0,0 +1,51 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.model.LatLng;
+
+/**
+ * A GeoJsonPoint geometry contains a single {@link com.google.android.gms.maps.model.LatLng}.
```

Comment:
Adding this to the GeoJsonGeometry interface as it applied to all geometry objects.